packages/compose/src/hooks/use-copy-to-clipboard/index.js

```diff
@@ -37,11 +44,11 @@ export default function useCopyToClipboard( text, onSuccess ) {
 	const onSuccesRef = useUpdatedRef( onSuccess );
 	return useRefEffect( ( node ) => {
 		// Clipboard listens to click events.
-		const clipboard = new Clipboard( node, {
+		const clipboard = new Clipboard( /** @type {Element} */ ( node ), {
```

Comment:
Is the cast from `Node` to `Element` safe?

Reply:
Actually I think the type of `node` should just be `HTMLElement` to begin with, making it unnecessary.

Reply:
Okay actually I went ahead and made `useRefEffect` generic, defaulting to `Node` but making it possible to make it `HTMLElement` for this function. This should be the safest option 🙂